vscode-dotty/package-lock.json

```diff
@@ -92,7 +92,7 @@
       "integrity": "sha1-mjRBDk9OPaI96jdb5b5w8kd47Dk=",
       "dev": true,
       "requires": {
-        "array-uniq": "^1.0.1"
+        "array-uniq": "1.0.3"
```

Comment:
I think your npm is doing something weird because mine is reverting these changes. Try upgrading to the latest npm (6.4.1)